packages/@ember/-internals/metal/lib/tracked/debugging.ts

```diff
@@ -0,0 +1,110 @@
+import { Tag, CONSTANT_TAG, UpdatableTag } from '@glimmer/reference';
+
+import { Tracker } from './tracker';
+
+type Option<T> = T | null;
+
+Ember.EMBER_DEBUG = {};
+
+let TAG_MAP = new Map<Tag | UpdatableTag, any>();
+
+interface TrackerHistory {
+  property: string;
+  value: any;
+  tracker: Tracker;
+}
+
+interface DebugTracking {
+  history: TrackerHistory[];
+  stacks: Set<Tracker>;
+}
+
+interface TrackerSnapshot {
+  tags: Array<Tag | UpdatableTag>;
+}
+
+interface TrackerProperty {
+  propertyName: string;
+  objectName: string;
+  objectRef: object;
+  tag: Tag | UpdatableTag;
+}
+
+Ember.EMBER_DEBUG.TRACKING = {
+  history: [],
+  stacks: new Set<Tracker>(),
+} as DebugTracking;
+
+function normalizeTrackerEntry(tag: any) {
+  // it looks like there are two types of data on a tracker's tag list.
+  // one type has the shape (which maybe isn't a tag?):
+  //   key: string
+  //   ref: object
+  //   tag: object
+  // the other is an actual Tag,
+  //   but with a private secret property _propertyName
+
+  let hostObject = TAG_MAP.get(tag);
+
+  let result: Partial<TrackerProperty> = {
+    objectName: hostObject && hostObject.__proto__.constructor.name,
+    objectRef: hostObject,
+    tag,
+  };
+
+  result.propertyName = tag.key || tag._propertyKey;
+
+  return result;
+}
+
+function prettyPrintTrackingInfo() {
+  let history = Ember.EMBER_DEBUG.TRACKING.history;
+
+  console.log(history, TAG_MAP);
+  history.forEach((tracker: TrackerSnapshot, i: number) => {
+    // tags seems to always be a touple
+    // (property that changed, the source property)
+    let values = tracker.tags;
+
+    let first = values[0];
+    let last = values[values.length - 1];
+
+    console.log(
+      '\t',
+      i,
+      normalizeTrackerEntry(first),
+      'Because',
+      normalizeTrackerEntry(last),
+      'changed'
+    );
+  });
+}
+
+Ember.EMBER_DEBUG.TRACKING.print = prettyPrintTrackingInfo;
+
+export function debugTracker(current: Tracker, parent: Option<Tracker>) {
+  // Convert the Tracker to an isolated moment in time
+  Ember.EMBER_DEBUG.TRACKING.history.push({
+    // hack around tags being a private field
+    tags: Array.from((current as any).tags.values()),
+  });
+
+  console.log('[ track event ] current:', current, 'parent:', parent);
+  // Ember.EMBER_DEBUG.TRACKING.print();
+}
+
+
+export function debugConsume(tracker: Tracker, tag: Tag | UpdatableTag, target: any) {
```

Comment:
the tracker itself doesn't have much information on it.
It has a tags Set, and that's about it
![image](https://user-images.githubusercontent.com/199018/65378217-63850800-dc83-11e9-9e44-d6b9b0491371.png)